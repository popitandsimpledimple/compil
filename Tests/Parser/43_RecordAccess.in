var point: record x, y: Integer; end;
begin
    point.x := 123;
    point.y := point.x;
    writeln(point.y, point.x);
end.