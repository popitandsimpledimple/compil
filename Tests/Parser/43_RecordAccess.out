program
   decls
      vars
         var
            point
               record
               params
                     x
                     type
                        integer
                     y
                     type
                        integer
            no expression
   block
      compounds statements
         :=
            record access
               point
               x
            123
         :=
            record access
               point
               y
            record access
               point
               x
         call
            writeln
               record access
                  point
                  y
               record access
                  point
                  x
